### PR TITLE
modified some things

### PR DESCRIPTION
**Pull request by ikenchina**
modified this library in order to adapt our requirement.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -94,17 +96,23 @@ func (c *Client) Production() *Client {
...
+	if n == nil {
+		//slog.Err("nil push notification")
+		return nil, errors.New("nil push notification")
+	}
 	payload, err := json.Marshal(n)
 
 	if err != nil {
 		return nil, err
 	}
+	slog.Info("APNS2 PUSH BODY: %s", payload)
```
Comment (henryren) [inline client.go:108]: It's better to remove info log from this package.

File: pushnotification.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package apns2
+
+import "github.com/ikenchina/apns2/payload"
```
Comment (henryren) [inline pushnotification.go:3]: This is wrong.

**Review by henryren — CHANGES_REQUESTED** [review]

**Comment by henryren** [comment]
Is it possible to use this package directly without doing any changes in it?

File: pushnotification.go (hunk 2 above)
Comment (ikenchina) [reply pushnotification.go:3]: sorry, forgot it.

File: client.go (hunk 1 above)
Comment (ikenchina) [reply client.go:108]: why need remove it?

**Pull request closed without merging**